packages/server/lib/browsers/index.js

```diff
@@ -195,7 +197,12 @@ module.exports = {
         // the browser opening
         return Promise.delay(1000)
         .then(() => {
+          if (instance === null) {
```

Comment:
When is instance ever null here?

Also, why do we need to return early? Is that because `onBrowserOpen` makes no sense if there is no instance, which refers to the current browser?

Reply:
This function takes a long time to resolve, enough time for the user to either switch to another browser or close the browser that just opened. In either case, the instance can become null so we don't want to run anything else.